internal/gps/identifier.go

```diff
@@ -40,6 +40,11 @@ import (
 // the type system when a path-ish string must have particular semantics.
 type ProjectRoot string
 
+// ProjectRootPtr takes a ProjectRoot value and returns a pointer.
+func ProjectRootPtr(p ProjectRoot) *ProjectRoot {
```

Comment:
if we must have this (and i'm not clear on why we do), can it at least not be exported?